qcfractal/server.py

```diff
@@ -544,17 +548,16 @@ def update_public_information(self) -> None:
         """
         Updates the public information data
         """
-        data = self.storage.get_server_stats_log(limit=1)["data"][0]
-
-        update = {
-            "counts": {
-                "collection": data["collection_count"],
-                "molecule": data["molecule_count"],
-                "result": data["result_count"],
-                "kvstore": data["kvstore_count"],
-            }
-        }
+        data = self.storage.get_server_stats_log(limit=1)["data"]
+
```

Comment:
do you need a default value for kvstore here?

Reply:
Yup.